client/components/auth/Login.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+// import { NavLink } from 'react-router-dom';
+import Popup from 'reactjs-popup';
+import Signup from './Signup';
+
+const Login = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const onLoginSubmit = async (e) => {
+    e.preventDefault();
+
+    return await axios
```

Comment:
`return await` is meaningless and serves no purpose. It is the same as `return ${anyPromise}` which is what you're doing here.